pulsar-broker-common/src/main/java/org/apache/pulsar/broker/resources/NamespaceResources.java

```diff
@@ -215,6 +258,17 @@ public boolean partitionedTopicExists(TopicName tn) throws MetadataStoreExceptio
                     tn.getEncodedLocalName()));
         }
 
+        public CompletableFuture<Void> clearPartitionedTopicMetadataAsync(NamespaceName namespaceName) {
+            final String globalPartitionedPath = joinPath(PARTITIONED_TOPIC_PATH, namespaceName.toString());
+
+            return existsAsync(globalPartitionedPath).thenAccept(exist -> {
```

Comment:
```suggestion
            return existsAsync(globalPartitionedPath).thenCompose(exist -> {
```

Reply:
In here, we don’t need a return value, so using `thenAccpet` is OK.

Reply:
If you use `thenCompose` here, we still need to deal with the return value, their logic is the same